clients/desktop/src/vault/components/VaultPage.tsx

```diff
@@ -1,4 +1,5 @@
 import { useCoreNavigate } from '@core/ui/navigation/hooks/useCoreNavigate'
+import { PageHeaderToggleTitle } from '@core/ui/page/PageHeaderToggleTitle'
```

Comment:
Please move PageHeaderToggleTitle to lib folder

Reply:
Fixed in the last commit